### DISCUSSION TITLE
Is inputting a Transform pipeline in the Dataset really data augmentation?

### DISCUSSION THREAD

**Discussion by ialvata** [mrdbourke/pytorch-deep-learning#722] [Q&A]
I've been experimenting with Transforms, and it seems that when we pass a picture to a transform like RandomPerspective, or Resize, it only outputs 1 picture, where the original is lost.

In data augmentation, we want to train our model on the original one plus the extra transformed one. Right now, the transform pipeline only outputs explicitly the transformed data, i.e. we don't train the model on the original.

Are we losing important information this way? 

Shouldn't we find a way to adapt the transform pipeline to output all the data?

**Comment by lboccato86** [comment]
I've just watched video 162 (https://www.learnpytorch.io/04_pytorch_custom_datasets/#93-construct-and-train-model-1) where Model 1 is trained, which stands for the same TinyVGG architecture, but using a different set of transforms that includes `TrivialAugmentWide()`, and I noticed that, in the end, we didn't actually explored `data_augmentation`, since the training set is comprised of transformed versions of the original images, but **does not contain the original images themselves**. Hence, the number of training samples remains the same. 

I think a nice experiment to be included (or, at least, suggested) in the course would be training Model 1 with the original images and the transformed images combined in an augmented set.
